src/main/kotlin/com/bardsoftware/backend/fleet/rmarkdown/Receivers.kt

```diff
@@ -120,23 +119,20 @@ open class TaskReceiver(tasksDirectory: String,
         var isPublished = true
 
         val compiledBytes: ByteString
-        val outputFileName: String
 
-        if (request.compiler == CompilerFleet.Compiler.MOCK) {
-            compiledBytes = ByteString.copyFrom(MOCK_PDF_BYTES)
-            outputFileName = MOCK_FILE_NAME
+        compiledBytes = if (request.compiler == CompilerFleet.Compiler.MOCK) {
```

Comment:
You can inline initialization into declaration one line above

Reply:
Done